local/config/b0magicBlock_4_miners_1_sharder.tmp.json

```diff
@@ -0,0 +1,241 @@
+{
+  "hash": "5337382eb82ccec9d6e61030d97091ff798582e28f1a33af254af6cbed78441d",
+  "previous_hash": "",
+  "magic_block_number": 1,
+  "starting_round": 0,
+  "miners": {
+    "type": 0,
+    "nodes": {
+      "31810bd1258ae95955fb40c7ef72498a556d3587121376d9059119d280f34929": {
+        "id": "31810bd1258ae95955fb40c7ef72498a556d3587121376d9059119d280f34929",
+        "version": "",
+        "creation_date": 0,
+        "public_key": "255452b9f49ebb8c8b8fcec9f0bd8a4284e540be1286bd562578e7e59765e41a7aada04c9e2ad3e28f79aacb0f1be66715535a87983843fea81f23d8011e728b",
+        "n2n_host": <Enter
```

Comment:
Is this placeholder intended to be here or forgotten to change?

Reply:
Place holder for you to put in your own IP addresses

Reply:
That's fine actually, good approach for making a user aware of placeholder.
Maybe only would be good to make it single-line, but not so important.
Just wanted to make sure that it isn't something accidental.